### ISSUE TITLE
Messagebox object support

### ISSUE THREAD

**Issue by BartaBlzs** [dotnet/wpf#10212]
Would be great if the MessageBox class had a method, that enables the developer to show not just strings, but any object.

**Comment by miloush** [comment]
I don't see any benefit to this, call `.ToString()` on the argument.

**Comment by h3xds1nz** [comment]
I agree with @miloush here, there's doesn't seem to be a visible benefit (but you may show one using an actual API suggestion).

Plus, it's a message box; it should serve for displaying messages, generally that's not a `ToString` override on an object.

**Comment by miloush** [comment]
@omarion you could and it would not, there is no reason why it should
